pkgs/development/python-modules/ibis-framework/default.nix

```diff
@@ -139,13 +139,28 @@ buildPythonPackage rec {
     # tries to download duckdb extensions
     "--deselect=ibis/backends/duckdb/tests/test_register.py::test_register_sqlite"
     "--deselect=ibis/backends/duckdb/tests/test_register.py::test_read_sqlite"
+    # all the following tests are fixed in the next release (6.0.0)
+    "--deselect=ibis/backends/tests/test_export.py::test_column_to_pyarrow_table_schema"
+    "--deselect=ibis/backends/tests/test_export.py::test_table_to_pyarrow_batches"
+    "--deselect=ibis/backends/tests/test_export.py::test_column_to_pyarrow_batches"
+    "--deselect=ibis/backends/tests/test_export.py::test_to_pyarrow_batches_memtable"
+    "--deselect=ibis/backends/tests/test_export.py::test_table_to_pyarrow_table_schema"
+    "--deselect=ibis/backends/tests/test_export.py::test_to_pyarrow_batches_borked_types"
+    "--deselect=ibis/backends/tests/test_export.py::test_table_pyarrow_batch_chunk_size"
+    "--deselect=ibis/backends/tests/test_export.py::test_column_pyarrow_batch_chunk_size"
+    "--deselect=ibis/backends/tests/test_export.py::test_roundtrip_partitioned_parquet"
+    "--deselect=ibis/tests/sql/test_sqlalchemy.py::test_order_by"
+    "--deselect=ibis/tests/sql/test_sqlalchemy.py::test_no_cart_join"
+    "--deselect=ibis/backends/tests/test_temporal.py::test_temporal_binop"
+    "--deselect=ibis/backends/tests/test_temporal.py::test_interval_literal"
   ];
 
   # patch out tests that check formatting with black
   postPatch = ''
     find ibis/tests -type f -name '*.py' -exec sed -i \
```

Comment:
```suggestion
  postPatch = ''
    # patch out tests that check formatting with black
    find ibis/tests -type f -name '*.py' -exec sed -i \
```

Reply:
I'd prefer not to add a comment in a way that affects the output if I change it. I'll leave this as is.

Reply:
I don't think it's a big deal.